docker-hub/registry-access-management.md

```diff
@@ -0,0 +1,114 @@
+
+## Configure Registry Access Management permissions
+
+To configure Registry Access Management permissions, perform the following steps:
+
+1. Sign into your [Docker Hub](https://hub.docker.com) account as an organization owner.
+2. Select an organization, navigate to the **Settings** tab on the **Organizations** page and click **Org Permissions**.
+3. Enable Registry Access Management to set the permissions for your registry.
+
+> **Note**
+>
+> When enabled, the Docker Hub registry is set by default, however you can also restrict this registry for your developers.
+
+4. Click **Add** and enter your registry details in the applicable fields, and click **Create** to add the registry to your list.
+5. Verify that the registry appears in your list and click **Save & Apply**.  You can verify that your changes are saved in the Activity tab.
+
+  > **Note**
+  >
+  > Once you add a registry, it can take a maximum of 24 hours for the changes to be enforced on your developers’ machines. If you want to apply the changes sooner, you must force a Docker logout on your developers’ machine and have the developer re-authenticate for Docker Desktop.  Also, there is no limit on the number of registries you can add. See the Caveats sections to learn more about limitations  when using this feature.
+
+### Enforce authentication
+
+To ensure that each org member uses Registry Access Management on their local machine, you can perform the following steps below to enforce sign-in under your organization. To do this:
+
+1. Download the latest version of Docker Desktop, and then
+2. Create a `registry.json` file.
+
+Download Docker Desktop 4.5 or a later release.
+
+- [Download and install for Windows](/desktop/windows/install/)
+- [Download and install for Mac](/desktop/mac/install/)
+
+#### Create a registry json file
+
+Before you create a `registry.json` file, ensure that the developer is a member of at least one organization in Docker Hub. If the registry.json file matches at least one organization the developer is a member of, they can sign into Docker Desktop and access all of their organizations.
+
+**On Windows**
+
+On Windows, you must create a file `C:\ProgramData\DockerDesktop\registry.json` with file permissions that ensure that the developer using Docker Desktop cannot remove or edit the file (that is, only the system administrator can write to the file). The file must be `JSON` and contain one or more organization names in the `allowedOrgs` key.
+
+To create your `registry.json` file on Windows:
```

Comment:
I'm not a PowerShell expert, so I might be wrong, but I'm confused by these commands. Step 3 doesn't seem to make sense and what does Step 4 do? Have you tested these instructions?

Reply:
Oh, I think I see what's happened now. The instructions at https://docs.docker.com/docker-hub/configure-sign-in/#create-a-registryjson-file make more sense. It looks like this copy has either been mis-edited from there, or the ones there have received later corrections.

Reply:
It must be "exactly one organization name" in the `allowedOrgs` key. We had to remove the ability to list multiple organizations there. Unfortunately the value is still syntactically a list (but now a list with exactly one element)

Reply:
Hey @stephen-turner Updated. Aligned with the testing doc, as well as the create-a-registryjson-file link in Docs